signet.gemspec

```diff
@@ -28,7 +28,7 @@ Gem::Specification.new do |gem|
   gem.add_runtime_dependency "jwt", ">= 1.5", "< 3.0"
   gem.add_runtime_dependency "multi_json", "~> 1.10"
 
-  gem.add_development_dependency "google-style", "~> 0.2"
+  gem.add_development_dependency "google-style", "~> 0.3.0"
```

Comment:
Unless you want to prevent automatic updates to `0.4`, I think it would be better to stick with the minor version `x.y` for convenience and use `0.3` here.